handlers/new-product-api/src/main/scala/com/gu/newproduct/api/addsubscription/Handler.scala

```
@@ -85,13 +88,15 @@ object Steps {
   ): ApiGatewayOp[Operation] =
     for {
       zuoraIds <- ZuoraIds.zuoraIdsForStage(stage).toApiGatewayOp(ApiGatewayResponse.internalServerError _)
-      zuoraConfig <- {
-        val loadConfig = LoadConfigModule(stage, fetchString)
-        loadConfig[ZuoraRestConfig].toApiGatewayOp("load zuora config")
-      }
+      loadConfig = LoadConfigModule(stage, fetchString)
```

I decided to fit in the existing way this lambda loads config (S3) but I think the more modern standard is some kind of parameter store or secrets manager.

This lambda needs the config in PROD before it can be merged (the whole lambda will fail fast if it's not valid - maybe it should be more lenient)